### PR TITLE
Add additional notes to the doc

### LINKED ISSUE #32143: OIDC document shoud update
### Describe the bug

https://quarkus.io/guides/security-openid-connect#testing-the-application
`username=alice&password=alice&grant_type=password` should change to `username=alice&password=alice&grant_type=password&scope=openid`

Or the token response scope will be only with "profile email". No "openid". Then the authorization will fail.

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by SetoKaiba**
Add additional notes to the doc. 

- Fixes: #32143

### CODE REVIEW

**Comment by sberyozkin** [comment]
Thanks very much @SetoKaiba for proposing this PR. Can I suggest to reword both Notes a little bit, to something like:

```
When the `quarkus.oidc.authentication.user-info-required` property is set to `true` to require that an access token is used to request `UserInfo`, you must add a `scope=oidc` query parameter to the token grant request command, for example:

[source,bash]
----
export access_token=$(\
    curl --insecure -X POST http://localhost:8180/realms/quarkus/protocol/openid-connect/token \
    --user backend-service:secret \
    -H 'content-type: application/x-www-form-urlencoded' \
    -d 'username=alice&password=alice&grant_type=password&scope=oidc' | jq --raw-output '.access_token' \
 )
----
```

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc [hunk 1]
```diff
@@ -334,6 +334,12 @@ export access_token=$(\
  )
 ----
 
+[NOTE]
+====
+If the setting below is set, you should add the openid to the scope in the query parameters.
```
Comment (gsmet) [inline docs/src/main/asciidoc/security-keycloak-authorization.adoc:339]: Which `setting below`? I wasn't able to figure out what this note was about.

Also maybe `user info` would be better than `openid`?

**Review by gsmet — COMMENTED** [review]
Thanks for the PR! I added a comment but don't hesitate to push back as I'm not very familiar with this area!

**Comment by SetoKaiba** [comment]
> Thanks very much @SetoKaiba for proposing this PR. Can I suggest to reword both Notes a little bit, to something like:
> 
> ```
> When the `quarkus.oidc.authentication.user-info-required` property is set to `true` to require that an access token is used to request `UserInfo`, you must add a `scope=oidc` query parameter to the token grant request command, for example:
> 
> [source,bash]
> ----
> export access_token=$(\
>     curl --insecure -X POST http://localhost:8180/realms/quarkus/protocol/openid-connect/token \
>     --user backend-service:secret \
>     -H 'content-type: application/x-www-form-urlencoded' \
>     -d 'username=alice&password=alice&grant_type=password&scope=oidc' | jq --raw-output '.access_token' \
>  )
> ----
> ```

I'll follow your instructions. But the scope should be openid instead of oidc. Right?

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc (hunk 1 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc:339]: Right, I agree, this is why I proposed a rewording, see my suggestion above

**Comment by sberyozkin** [comment]
@SetoKaiba 

>But the scope should be openid instead of oidc. Right?

Yeah, good catch, thanks :-)

**Review by sberyozkin — APPROVED** [review]
LGTM, thanks @SetoKaiba 

Guillaume, FYI, hopefully it is clearer now

**Pull request merged**